docs/src/tutorial/simple_system.md

```diff
@@ -138,6 +182,24 @@ If you are not sure whether you have the latest version, please upgrade to ensur
     release energy into the *electricity\_node*.
 ![image](../figs_simple_system/simple_system_add_unit__to_node_relationships.png)
 
+-   Right click on the *unit\_\_node\_\_node* class, and select
+    *Add relationships* from the context menu. The *Add relationships*
+    dialog will pop up.
+
+-   For each of the units enter the unit under *unit*,
+    *electricity_node* under the first node and *fuel_node* under the 
+    second node. These relationships will define the relation (or behavior)
+     between the output and input of the unit.
+![image](../figs_simple_system/simple_system_add_unit__node__node.png)
+
+!!! note
+    The *unit\_\_node\_\_node* relationship is necessary to bind the flow 
```

Comment:
Set a boundary for the flow? "Binding" sounds weird to me, but maybe it's just a term I haven't heard yet.

Reply:
I'm going to trust your english more than mine.